scripts/us_bls/jolts/bls_jolts.py

```diff
@@ -0,0 +1,285 @@
+    seasonal_adjustment = row['seasonal_adjustment']
+
+    return f"dcs:{base_stat_var}_NAICS_{industry_code}_{seasonal_adjustment}"
+
+  # Build map to Statistical Variable
+  jolts_df['seasonal_adjustment'] = jolts_df['seasonal'].apply(lambda adjustment: "Adjusted" if adjustment == "S" else "Unadjusted")
+  jolts_df['StatisticalVariable'] = jolts_df.apply(map_row_to_statistical_variable, axis=1)
+  jolts_df['Value'] = jolts_df['value']
+
+  return jolts_df, schema_mapping
+
+
+def create_statistical_variables(jolts_df, schema_mapping):
+  """ Creates Statistical Variable nodes.
+
+    A new statistical industry is needed for each of the 6 job variables and for every industry
+    The industry codes may be either NAICS or BLS_JOLTS aggregations. The schema_mapping is used 
+    for additional information for each of the 6 job variables. These new variables are written
+    to the statistical variables mcf file.
+
+    Args:
+      jolts_df: The dataframe of BLS Jolts data created by generate_cleaned_dataframe
+      schema_mapping: The schema mapping created by generate_cleaned_dataframe
+  """
+
+  template_stat_var = """
+  Node: dcid:{STAT_CLASS}_NAICS_{INDUSTRY}_{ADJUSTMENT}
+  typeOf: dcs:StatisticalVariable
+  populationType: {POPULATION}
+  jobChangeEvent: dcs:{JOB_CHANGE_EVENT}
+  statType: dcs:measuredValue
+  measuredProperty: dcs:count
+  measurementQualifier: {BLS_ADJUSTMENT}
+  naics: dcid:NAICS/{INDUSTRY}
+  """
+
+  # Map industry and seasonal adjustment to statistical variable name
+  adjustment_types = [("Adjusted", "dcs:BLSSeasonallyAdjusted"), ("Unadjusted", "dcs:BLSSeasonallyUnadjusted")]
+
+  # Output the schema mapping to a new file
+  with open(STATISTICAL_VARIABLE_FILE, "w+", newline="") as f_out:
+    for schema_name, pop_type, job_change_event, _ in schema_mapping:
+
+        unique_industries = list(jolts_df['industry_code'].unique())
+
+        for industry_code in unique_industries:
+          for adjusted_dcid_map, adjusted_schema in adjustment_types:
+            # Create new schema object
+            stat_var_schema = textwrap.dedent(template_stat_var)
+
+            # Remove separation type entry if not includes
+            if job_change_event == "":
+              stat_var_schema = stat_var_schema.replace("jobChangeEvent: dcs:{JOB_CHANGE_EVENT}\n", "")
+
+            # Replace all other fields
+            stat_var_schema = stat_var_schema.replace("{STAT_CLASS}", schema_name)   \
+                                            .replace("{INDUSTRY}", industry_code)   \
+                                            .replace("{ADJUSTMENT}", adjusted_dcid_map)   \
+                                            .replace("{BLS_ADJUSTMENT}", adjusted_schema)   \
+                                            .replace("{POPULATION}", pop_type)      \
+                                            .replace("{JOB_CHANGE_EVENT}", job_change_event)  
+
+            f_out.write(stat_var_schema)
```

Comment:
One can just f_out.write the previously assigned value.

Reply:
I prefer this for debugging. I can update it if you feel strongly about the change.